annif/project.py

```diff
@@ -62,10 +62,13 @@ def _init_access(self):
                 project_id=self.project_id)
 
     def _initialize_analyzer(self):
-        analyzer = self.analyzer
-        logger.debug("Project '%s': initialized analyzer: %s",
-                     self.project_id,
-                     str(analyzer))
+        try:
+            analyzer = self.analyzer
+            logger.debug("Project '%s': initialized analyzer: %s",
+                         self.project_id,
+                         str(analyzer))
+        except AnnifException as err:
```

Comment:
Can we avoid having to catch the exception here? Not sure it's a great idea that simply accessing `self.analyzer` may trigger an exception. Returning None is a bit more elegant, but then of course the missing analyzer case has to be handled by the calling code.